posthog/models/test/test_person_override_model.py

```diff
@@ -1,11 +1,15 @@
+import contextlib
```

Comment:
these changes should be in https://github.com/PostHog/posthog/pull/14477

Reply:
I've moved these changes to #14477 and rebased this PR.